pluginconfig/commit.go

```diff
@@ -33,6 +33,13 @@ type CommitPluginConfig struct {
 
 	// OffchainConfig is the offchain config set for the commit DON.
 	OffchainConfig CommitOffchainConfig `json:"offchainConfig"`
+
+	// RMNEnabled is a flag to enable/disable RMN signature verification.
+	RMNEnabled bool `json:"rmnEnabled"`
+
+	// RMNSignaturesTimeout is the timeout for RMN signature verification.
+	// Typically set to `MaxQueryDuration - e`, where e some small duration.
+	RMNSignaturesTimeout time.Duration `json:"rmnSignaturesTimeout"`
```

Comment:
Would it be simpler to set the `e` here instead? Cuz the `ctx` for query will already be a `context.WithTimeout(MaxQueryDuration)`, so you can do something like:

```golang
// in query 
queryDeadline := ctx.Deadline()
// set earlier deadline for sig fetching
sigDeadline := queryDeadline.Sub(p.offchainConfig.RMNQueryE)
ctx, cancel := context.WithDeadline(ctx, sigDeadline)
defer cancel()
// use ctx...
```

Reply:
Yeah maybe safer also. I'll do it next.